### DISCUSSION TITLE
首次在Win11上使用软件

### DISCUSSION THREAD

**Discussion by TotoWang-hhh** [DearVa/ExplorerEx#33] [Ideas]
# 下面是一些使用体验的说明

**首先是提醒作者大大注意休息啊！我一初中生都可珍惜睡眠时间了呢！**

![image](https://user-images.githubusercontent.com/67090055/166250197-ccd0f227-ab50-408a-9bd6-7a089ebc0fd2.png)

---

## 概要
我并没有觉得软件在Win11上表现非常好。
我为了获得我认为的最佳体验，修好了我身边唯一一台Win11电脑。但是最终获得的体验令我大大失望。

## 兼容性问题
Win11似乎仍然没有预装ExplorerEx必须的NET6.0，以至于我需要再次去微软的官网下载，忍受那仅次于百度网盘的下载速度……
可见ExplorerEx是有多挑食！

## 模糊效果的问题
之前告诉过我窗口背景使用了云母效果，但是调用了Win11的API，所以Win10没有。但是现在我用了Win11，窗口背景模糊仍然没有。

## 标签页拖动问题
根据Release中的描述，这个功能还在重构 <del>（但是你以为我会放过它？）</del>
虽说这可以让我原谅它还会报错的BUG，但是我还是要吐槽一下：作者您为啥不直接暂时禁用这个功能，而是让我们**看着这玩意弹出一个巨大的错误框，然后整个软件直接卡退**？

顺便提醒下大家：在同一窗口里拖动标签页的功能是正常的，可以进行标签页排序。

---

嗯……好像没啥了

**Comment by DearVa** [comment]
很抱歉给您带来了不好的体验，目前软件还在不断开发中（这几天要赶一个ddl暂时搁置了），我会不断努力。
兼容性问题：我在下次发布时会将运行时一并打包，不必担心兼容性问题（只要是win7 x64以上电脑均可运行）。
模糊效果：云母效果并不是单纯的模糊，它是一种不太明显的渐变色，根据您的壁纸来取样。
标签页拖动：我遇到了些问题，需要较长的时间来重构它，我会在下次暂时屏蔽这个功能。

**Reply by TotoWang-hhh** [reply]
需要看winver吗？嗯，需要
![image](https://user-images.githubusercontent.com/67090055/166258006-4b5cd5da-d5df-4370-a244-5a61393ee4f9.png)
需要看软件效果吗？嗯嗯，也需要
![image](https://user-images.githubusercontent.com/67090055/166258253-56ca0cab-6d2f-4d0f-a5df-7c45bb10b564.png)

不是我不懂云母，而是我这里连渐变都没有，直接就是白背景
---
我直接在Reply里面重发一遍，我相信你在线：）

**Comment by TotoWang-hhh** [comment]
需要看winver吗？嗯，需要
![image](https://user-images.githubusercontent.com/67090055/166258006-4b5cd5da-d5df-4370-a244-5a61393ee4f9.png)
需要看软件效果吗？嗯嗯，也需要
![image](https://user-images.githubusercontent.com/67090055/166258253-56ca0cab-6d2f-4d0f-a5df-7c45bb10b564.png)

不是我不懂云母，而是我这里连渐变都没有，直接就是白背景

**Comment by TotoWang-hhh** [comment]
# 如果您觉得还不够明显，我可以给您看看Edge（有靠谱的云母效果，请注意标题栏）和ExplorerEx（没有或看不出云母效果）的对比图：
![image](https://user-images.githubusercontent.com/67090055/166259929-c5a3a088-de3e-4d5e-b50f-67d6c56f9e82.png)
![image](https://user-images.githubusercontent.com/67090055/166259943-ba6e44d7-0446-45f0-b609-89c4355201d5.png)

**Comment by TotoWang-hhh** [comment]
# 补充：您没有必要为了一个拒绝访问的错误弹出大量的报错信息！
事情是这样的：
我试图打开我的EFI分区，在看见一个看上去像刻意设计的的错误框之后，我看到了……
![image](https://user-images.githubusercontent.com/67090055/166261003-54f89f6e-d3d9-447b-878c-2201b18454d5.png)

## 然而，ExplorerEx似乎仍然没有放弃加载这个已经被告知拒绝访问的磁盘：
![image](https://user-images.githubusercontent.com/67090055/166261125-acae3656-40ce-4ee8-8272-71a6ec972e13.png)

---

# 这时最有趣的事发生了
你无法切换到别的位置！
